Gemfile

```diff
@@ -16,7 +16,6 @@ end
 
 group :test do
   gem 'minitest'
-  gem 'minitest-reporters'
```

Comment:
I think keeping this is good, as these reporters are useful for more than just Jenkins. @ekohl ?

Reply:
Should I keep it now @ehelms @ekohl? I don't see any other configuration for it to use, other than Jenkins in this project.

Reply:
It slightly changes the output and gives a progress bar but I'm not sure I ever found that very useful. I'm fine with either way.